I've added the 404

gavinc-emc opened the pull request.

gavinc-emc committed:
404 page added

gavinc-emc committed:
error.php file wierdness

andyt-emc commented:
@gavinc-emc It looks like the .less files are in the wrong folder.  These need to be in 'assets' and imported into the template.less file.

andyt-emc reviewed (changes requested):
Move the less file into the assets folder please.

gavinc-emc committed:
Fix for assets folder location

Moved 404.less file, added import line to assets/template.less and removed old files/folders

andyt-emc reviewed (approved):
Happy the requested amends have been made. @gavinc-emc
